CRM/Report/Form/Mailing/Detail.php

```diff
@@ -131,6 +131,10 @@ public function __construct() {
           'title' => ts('Delivery Status'),
           'default' => TRUE,
         ],
+        'time_stamp' => [
+          'title' => ts('Delivery Date'),
+          'default' => TRUE,
```

Comment:
This default doesn't seem to work for me to make it checked by default, but the field itself seems fine.